ex/loqui/lib/loqui/protocol/frames.ex

```diff
@@ -1,6 +1,10 @@
-defmodule Loqui.Frames do
+defmodule Loqui.Protocol.Frames do
   use Loqui.{Opcodes, Types}
 
+  def hello(flags, payload_data) do
+    <<@opcode_hello, flags :: uint8, 1::uint8, byte_size(payload_data) :: uint32, payload_data :: binary>>
```

Comment:
use iodata when writing for less copying (nb could be `IO.iodata_length(payload_data)` if iodata support.)
```elixir
[@opcode_hello, flags,<<byte_size(payload_data) :: unit32>> | payload_data]
```